h2o-genmodel/src/main/java/hex/genmodel/tools/PrintMojo.java

```diff
@@ -28,11 +28,14 @@
   protected int treeToPrint = -1;
   protected int maxLevelsToPrintPerEdge = 10;
   protected boolean detail = false;
-  protected String outputFileName = null;
+  protected String destination = null;
   protected String optionalTitle = null;
   protected PrintTreeOptions pTreeOptions;
   protected boolean internal;
   protected final String tmpOutputFileName = "tmpOutputFileName.gv";
+  protected int nPlaces = -1;
```

Comment:
these are never used, only in runFromFlow which is a subclass, I think it would be better to create default PrintTreeOptions into a new constructor

Reply:
they are used (and can be set to something else by the user) in PrintMojo.java#parseArgs() in version for terminal